src/main/java/org/folio/rest/impl/FiscalYearsApi.java

```diff
@@ -85,4 +93,8 @@ public void deleteFinanceFiscalYearsById(String id, String lang, Map<String, Str
       .exceptionally(fail -> handleErrorResponse(handler, helper, fail));
   }
 
+  private void getFYearWithSeries(FiscalYear fiscalYear) {
```

Comment:
Could you rename this method since it does not return fiscalYear but set series.